gammapy/time/lightcurve.py

```diff
@@ -320,6 +321,136 @@ def create_fixed_time_bin(time_step, spectrum_extraction):
             ])
         return intervals
 
+    def create_fixed_significance_bin_lc_table(self, significance, significance_method, energy_range,
+                                               spectrum_extraction, separators=[]):
+        """
+
+        Create time intervals such that each bin of a light curve reach a given significance and store information
+
+        Parameters
+        ----------
+        significance : float
+            Target significance for each light curve point
+        significance_method : {‘lima’, ‘simple’}
+            Significance method (see `~gammapy.stats.significance_on_off`)
+        energy_range : `~astropy.units.Quantity`
+            True energy range to evaluate integrated flux (true energy)
+        spectrum_extraction : `~gammapy.spectrum.SpectrumExtraction`
+            Contains statistics, IRF and event lists
+        separators : `list` of float
+            Contains a list of time (in MJD) to break the current point creation
+
+        Returns
+        -------
+        table : `~astropy.table.Table`
+            Table of time intervals  and information about their content : on/off events, alpha, significance
+
+        Examples
+        -------
```

Comment:
There's a missing `_` here and the examples section isn't properly formatted. Put this:
```
        Examples
        --------
        Sometimes it can be useful to have the time intervals as a list of tuples
        of start and stop time. You can get that from the table like this::

            intervals = list(zip(table['t_start'], table['t_stop']))
```